### PR TITLE
Create a zh_CN langage

### PR DESCRIPTION
**Pull request by TartaricAcid**

### CODE REVIEW

**Comment by bonii-xx** [comment]
Cool, your language.lang file seems to have the wrong encoding though. Make sure it's saved correctly.

**Comment by TartaricAcid** [comment]
When I used **UTF-8** code language.lang file, it doesn't normally display in game.
Then I use **GB2312** , it can be used normally.
So the file look like have some problem in GitHub
But there is no problem in the game.

**Comment by TartaricAcid** [comment]
> GB2312 is the registered internet name for a key official character set of the People's Republic of China, used for simplified Chinese characters.
From wikipedia
[GB2312](https://en.wikipedia.org/wiki/GB_2312)

File: resources/assets/tconstruct/book/zh_CN/intro/toolstation.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "title":"工具装配台,
```
Comment (KnightMiner) [inline resources/assets/tconstruct/book/zh_CN/intro/toolstation.json]: Your JSON is missing a quotation mark here, that will most likely break things in game

File: resources/assets/tconstruct/lang/zh_CN.lang [hunk 2]
```diff
@@ -0,0 +1,832 @@
...
+gui.waila.casting.progress=进度: %s
+gui.waila.casting.recipe=配方: %s
+
+option.tconstruct.tank=储罐液体
+option.tconstruct.casting=铸造配方
+option.tconstruct.progress=制作进度
+
+
+# Config
+onfig.synced.restart=你的配置已经被更新以和服务器匹配.请重启Minecraft.
```
Comment (LovelyFlare) [inline resources/assets/tconstruct/lang/zh_CN.lang]: Should be config instead of "onfig"

File: resources/assets/tconstruct/lang/zh_CN.lang [hunk 3]
```diff
@@ -0,0 +1,832 @@
...
+modifier.haste.name=急速
+modifier.haste.name2=急速 II
+modifier.haste.name3=急速 III
+modifier.haste.name4=急速 IV
+modifier.haste.name5=急速 V
+mmodifier.haste.desc=§o哇啊啊啊啊啊！§r\n红石脉冲会使你的工具挖的更快.
+modifier.knockback.name=击退
+modifier.knockback.desc=§o本垒打！§r\n以一种很酷的方式教会怪物如何飞行.
+modifier.luck.name=时运
+modifier.luck.name2=时运
```
Comment (LovelyFlare) [inline resources/assets/tconstruct/lang/zh_CN.lang]: This line needs to be deleted

File: resources/assets/tconstruct/lang/zh_CN.lang [hunk 4]
```diff
@@ -0,0 +1,832 @@
...
+tile.tconstruct.seared_tank.window.name=焦黑视窗
+tile.tconstruct.seared_tank.tooltip=敲掉时也能保存其中的流体
+tile.tconstruct.seared_glass.glass.name=焦黑玻璃
+tile.tconstruct.seared_glass.glass.tooltip=可以做冶炼炉的材料，也是美观的建材\n不能存储液体
+
+tile.tconstruct.stone_torch.name=石柄火把
+tile.tconstruct.stone_ladder.name=石头楼梯
+tile.tconstruct.wood_rail.name=木质轨道
+tile.tconstruct.wood_rail_dropper.name=带活版门的木质轨道
+tile.tconstruct.wood_rail_dropper.tooltip=漏斗矿车可以\n穿过这个方块下面
```
Comment (LovelyFlare) [inline resources/assets/tconstruct/lang/zh_CN.lang]: Line 175 and 176 should be "tile.tconstruct.wood_rail_trapdoor.name" and "tile.tconstruct.wood_rail_trapdoor.tooltip"

File: resources/assets/tconstruct/book/zh_CN/intro/toolstation.json (hunk 1 above)
Comment (TartaricAcid) [inline resources/assets/tconstruct/book/zh_CN/intro/toolstation.json]: Thank you for pointing out the mistake.

File: resources/assets/tconstruct/lang/zh_CN.lang (hunk 4 above)
Comment (TartaricAcid) [inline resources/assets/tconstruct/lang/zh_CN.lang]: I'm sorry for my carelessness.
Thank you so much for pointing out so many mistakes.
And I feel surprised for your meticulous.
Thank you very much!  **╰(￣▽￣)╭**

**Comment by bonii-xx** [comment]
I still find it a bit weird that the encoding doesn't work for the .lang file but the others... on the other hand I'm not surprised by anything when it comes to Minecraft internals ;)
Looking good, thanks for the translation!

**Pull request merged**